plaid/payment_test.go

```diff
@@ -91,26 +91,39 @@ func TestPaymentWithBacsAndIban(t *testing.T) {
 	assert.NotNil(t, paymentRecipientGetResp.Address)
 
 	// testing common flows with the link_token
-	commonPaymentTestFlows(t, recipientID, true)
+	commonPaymentTestFlows(t, recipientID, true, false)
 
-	// testing commong flows with the legacy payment_token
-	commonPaymentTestFlows(t, recipientID, false)
+	// testing common flows with the legacy payment_token
+	commonPaymentTestFlows(t, recipientID, false, false)
+
+	// testing common flows with options
+	commonPaymentTestFlows(t, recipientID, false, true)
 }
 
-func commonPaymentTestFlows(t *testing.T, recipientID string, useLinkToken bool) {
+func commonPaymentTestFlows(t *testing.T, recipientID string, useLinkToken bool, useOptions bool) {
 	paymentRecipientListResp, err := testClient.ListPaymentRecipients()
 	assert.Nil(t, err)
 	assert.True(t, len(paymentRecipientListResp.Recipients) > 0)
 
+	var options PaymentOptions
```

Comment:
This should probably be a pointer... currently in the case of `false` this is using an empty struct, which I'm guessing isn't what we want tob e testing?